### PR TITLE
helm: fixes for "global.rbacEnabled=false"

### PR DESCRIPTION
**Pull request by kyessenov**
Signed-off-by: Kuat Yessenov <kuat@google.com>

### CODE REVIEW

File: install/kubernetes/helm/istio/templates/namespace.yaml [hunk 1]
```diff
@@ -1,4 +0,0 @@
-apiVersion: v1
-kind: Namespace
-metadata:
- name: {{ .Release.Namespace }}
```
Comment (kyessenov) [inline install/kubernetes/helm/istio/templates/namespace.yaml]: Not sure what to do about this problem...

**Comment by mandarjog** [comment]
@yusuoh if namespace resource is part of the yaml file, helm does not like it.
However it means that a user first needs to create a namespace and then run `kubectl apply -f filename`

File: install/kubernetes/helm/istio/templates/namespace.yaml (hunk 1 above)
Comment (kyessenov) [reply install/kubernetes/helm/istio/templates/namespace.yaml]: @yusuoh can you point me to the script that tries to run this without helm? helm does not like this particular template.

Comment (yusuoh) [reply install/kubernetes/helm/istio/templates/namespace.yaml]: @mandarjog @kyessenov 
It is in #4612. I moved the namespace template out of Helm charts and copy the content to the head of every generated yamls in the Makefile.

Comment (kyessenov) [reply install/kubernetes/helm/istio/templates/namespace.yaml]: It's still there after your PR: https://github.com/istio/istio/blob/master/install/kubernetes/helm/istio/templates/namespace.yaml

Comment (yusuoh) [reply install/kubernetes/helm/istio/templates/namespace.yaml]: Oh, the pr is not merged yet. I am waiting for the approval.

Comment (kyessenov) [reply install/kubernetes/helm/istio/templates/namespace.yaml]: Ok, whichever one gets first is not important then. This one is ready, too.

**Pull request merged**